level2/[level2] 정윤주/application/src/main/java/calculator/model/CalculatorModel.java

```diff
@@ -0,0 +1,72 @@
+package calculator.model;
+
+import java.util.ArrayList;
+
+public class CalculatorModel {
+    public int calculator(String input) {
```

Comment:
메소드의 이름은 동사로 사용하는 것이 좋아요. 행위를 나타내는 것이니까요!
```suggestion
    public int calculate(String input) {
```